rossum/tools/xls_to_csv.py

```diff
@@ -46,6 +46,7 @@ def cli(
         dtype=(str, str),
         header=header,
         skiprows=skiprows_list,
+        engine="openpyxl",
```

Comment:
I am slightly concerned about security - by no means I am oriented in excel files vulnerabilities, but according to `openpyxl` [docs](https://openpyxl.readthedocs.io/en/stable/) the library admits some. I will leave the assessment of severity to you - I guess that it should not matter much as the tool is used by the end-user for her/his own purposes and it is up to them to maintain some security model.